src/pages/signin/PasswordForm.js

```diff
@@ -35,28 +42,53 @@ const propTypes = {
 };
 
 const defaultProps = {
-    account: {},
+    account: {
+        password: '',
+        twoFactorAuthCode: '',
```

Comment:
Not gonna repeat myself here.. but this file has all the same issues as the other file. Will take a closer look once the first round of comments are resolved.